tafinder/app/models/term.rb

```diff
@@ -1,6 +1,6 @@
 class Term < ActiveRecord::Base
   has_many :rankings, dependent: :nullify
```

Comment:
Right now the DB looks like this:
Application <--- one-to-many --- Term
Application --- one-to-many ---> Ranking <--- one-to-many --- Term

I think this should change to: has_many :rankings, dependent: :nullify, through: :applications and then the term_id attribute should be removed from the ranking model. This would make the DB look like:
Application --- one-to-many ---> Term
Application --- one-to-one ---> Ranking